xarray/core/variable.py

```diff
@@ -335,7 +335,13 @@ def shape(self):
 
     @property
     def nbytes(self):
```

Comment:
```suggestion
    def nbytes(self) -> int:
```
Can we add the typing here as well?